.github/workflows/merge.yml

```diff
@@ -36,11 +36,7 @@ jobs:
       - run: npx nx-cloud start-ci-run
       - run: npx nx affected --target=build --parallel --max-parallel=3
       - name: semantic-release
-        if: github.ref == 'refs/heads/master'
         run: npm run nx affected -- --target=version
```

Comment:
we don't need the ref's with this because of the .nxreleaserc config has a semantic release branche setup